Dockerfile

```diff
@@ -15,42 +16,41 @@ RUN /bin/bash ./Miniconda-latest-Linux-x86_64.sh -b -p /home/rdkit/miniconda
 
 ENV PATH /home/rdkit/miniconda/bin:$PATH
 
-RUN conda update conda --yes --quiet
-RUN conda install jinja2 conda-build --yes --quiet
-RUN conda install anaconda-client --yes --quiet
+RUN \
+    conda update conda --yes --quiet &&\
+    conda install jinja2 conda-build --yes --quiet &&\
```

Comment:
some question as above, why not combine all the conda packages to be installed into one `conda install` command?